src/helpers/sendTweetOnDiscord.ts

```diff
@@ -23,10 +24,21 @@ export default async function ({
   const channel = await getChannel()
   const row = actionButtonBuilder({ blockchainId, contractAddress })
   const symbol = await getSymbol(derivativeAddress)
+  let english = true
```

Comment:
`english` here is a proxy for `translation`, let's do:
```TS
let translation = ''
reasons.split(', ').forEach(async (reason) => {
    if (reason.includes('not English')) 
      translation = await translate(text, { to: 'en' })
  })
```

Reply:
Then you can check do:
```TS
 if (translation)
   translation = '\nTranslation: ' + translation.text
 ```

Reply:
Doesn't seem to work because the `property doesn't exist on 'never'` shows up when I do
`let translation
  let translationText = ''
  reasons.split(', ').forEach(async (reason) => {
    if (reason.includes('not English')) {
      translation = await translate(text, { to: 'en' })
    }
  })
  if (translation) {
    translationText = '\nTranslation: ' + translation?.text
  }`

Reply:
Also tried different variations with 
`let translation: translate.ITranslateResponse`
`let translation: translate.ITranslateResponse | undefined`
`let translation: translate.ITranslateResponse | undefined = undefined`

and 
`if (translation != undefined)`

but doesn't seem to work